src/opensles/AudioOutputStreamOpenSLES.cpp

```diff
@@ -110,6 +110,34 @@ SLuint32 AudioOutputStreamOpenSLES::channelCountToChannelMask(int channelCount)
 }
 
 Result AudioOutputStreamOpenSLES::open() {
+    // Log unsupported attributes
+    // only report if changed from the default
+    // Usage::Media
+    // NOT TRUE! See OpenSLES_convertOutputUsage
+    // if (mUseage != Usage::Media) {
+    //    LOGW("Usage not supported on OpenSL ES streams.");
+    //}
+
+    // Content Type
+    if (mContentType != ContentType::Music) {
+        LOGW("ContentType [AudioStreamBuilder::setContentType()] "
+             "is not supported on OpenSLES output streams.");
+    }
+    // Sharing Mode
+    if (mSharingMode != SharingMode::Shared) {
+        LOGW("SharingMode [AudioStreamBuilder::setSharingMode()] "
+             "is not supported on OpenSLES output streams.");
+    }
+    // Performance Mode
+    if (mPerformanceMode != PerformanceMode::None) {
```

Comment:
Performance Mode is supported in N+
https://github.com/google/oboe/blob/6359dca09f7c91ebd8b92ffcc5738b041f685d12/src/opensles/AudioStreamOpenSLES.cpp#L166

Reply:
Is that documented? (Not in AudioStreamBase.h or Definitions.h).

Reply:
No. But it needs to be.